projects.yaml

```
@@ -480,3 +480,180 @@ projects:
       user_guide: https://openapi.docs.pyansys.com/version/stable/index.html
       examples: https://openapi.docs.pyansys.com/version/stable/index.html#brief-example
       api: https://openapi.docs.pyansys.com/version/stable/api/index.html
+
+  PyAEDT Common Toolkit:
```

Avoid using spaces. Let's go with hyphens as separators.

```suggestion
  pyaedt-common-toolkit:
```